dotcom-rendering/src/amp/components/Ad.tsx

```diff
@@ -28,6 +28,24 @@ const amazonConfig = {
 	aps: { PUB_ID: '3722', PARAMS: { amp: '1' } },
 };
 
+const mapAdTargeting = (adTargeting: AdTargeting): AdTargetParam[] => {
+	const adTargetingMapped: AdTargetParam[] = [];
```

Comment:
This is totally a nitpick, but maybe this should be `let` rather than `const` since it's going to have stuff pushed into it?

Reply:
It's not reassigned so I _think_ `const` is appropriate here!